js/components/portal-dashboard/custom-select.tsx

```diff
@@ -6,17 +6,18 @@ import { SvgIcon } from "../../util/svg-icon";
 import css from "../../../css/portal-dashboard/custom-select.less";
 
 interface IProps {
+  dataCy: string;
+  disableDropdown?: boolean;
+  HeaderIcon: SvgIcon;
+  isHeader?: boolean;
   items: SelectItem[];
   onSelectItem: (value: string) => void;
+  selectState?: string;
```

Comment:
As suggested below, I think the naming could be clearer here. If I were just naming this prop I'd probably name it `selectedItem` for consistency with `onSelectItem`, but that's not ideal because it's not really an item. If we think of this as a simple control like any other, e.g. a standard `<select>`, then the props should be named `value` and `onChange`. Then perhaps the fields of `SelectItem` should be named `value` and `label`. More broadly, I think there was a notion that `CustomSelect` was a component that could be standardized for use across multiple CC applications, but instead each usage has been customized in unique ways so that much of the benefit of consistency has been lost. Ideally, any changes to this component should be made with an eye toward general applicability rather than local expediency. I'm not sure how much of this is worth addressing now, but it's something to think about.

Reply:
I like these changes.  In making them, I would also vote to turn the state variable `selected` also into `value` so we have consistency with the prop.  I also agree on attempting to make things like the `CustomSelect` and the `ToggleControl` generic and reusable in other apps.  That might be a fun side-project to add them to the CC components library that others were working on:
https://github.com/concord-consortium/concord-react-components

Reply:
And in line with your naming ideas, I think we can turn `handleListClick` into `handleChange` for consistency.

Reply:
I made the above changes.